triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -723,6 +724,16 @@ private[lf] class Runner private (
         ctx.copy(value = converter.fromHeartbeat)
     }
 
+  private[this] def conflateMsgValue: TriggerContextualFlow[SValue, SValue, NotUsed] =
+    Flow
+      .fromFunction[TriggerContext[SValue], TriggerContext[SValue]](identity)
+      .conflateWithSeed { case ctx @ Ctx(_, value, _) =>
+        ctx.copy(value = BackStack.empty :+ value)
+      } { case (ctx @ Ctx(_, stack, _), Ctx(_, value, _)) =>
+        ctx.copy(value = stack :+ value)
+      }
+      .map(_.map(stack => SList(stack.toImmArray.toFrontStack)))
+
```

Comment:
@carlpulley-da could you clean this component. Here I just drop the context of the message except the first one. There is probably something nicer to do.